node/src/Transaction.ts

```diff
@@ -2942,11 +2942,11 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
      *     These IDs are deleted from the Pending Entries List.
      */
     public xautoclaimJustId(
-        key: string,
-        group: string,
-        consumer: string,
+        key: GlideString,
+        group: GlideString,
+        consumer: GlideString,
         minIdleTime: number,
-        start: string,
+        start: GlideString,
```

Comment:
I think this is alwyas a simple string

Reply:
perhaps, but this will need to be fixed in lots of places

Reply:
I fixed this in https://github.com/valkey-io/valkey-glide/pull/2287